t/misc_needles/test_tag1.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Could those needles be a bit more minimalized, e.g. remove things like clickpoint and other stuff that's not important for the actual test?